test/acts_as_authentic_test/session_maintenance_test.rb

```diff
@@ -92,5 +139,14 @@ def test_resetting_password_when_logged_out
       assert UserSession.find
       assert_equal ben, UserSession.find.record
     end
+
+    private
+
+    # This method makes sure that the config is always correct
+    # before each test
+    def setup_config(after_create = true, after_password_change = true)
+      User.log_in_after_create = after_create
+      User.log_in_after_password_change = after_password_change
+    end
```

Comment:
Hi @jaredbeck , sorry for misunderstanding. Well, some of the specs require log_in_after_create to be false, some other specs require log_in_after_password_change to be false and some other specs require the default configuration which is that both are true. For those that are expecting both to be true, I don't have to do anything because that's the default setup.